frontend/src/metabase/lib/utils.unit.spec.ts

```diff
@@ -0,0 +1,110 @@
+import MetabaseUtils from "metabase/lib/utils";
+
+describe("utils", () => {
+  describe("versionToNumericComponents", () => {
+    it("should pad to 4 numeric components", () => {
+      expect(
+        MetabaseUtils.versionToNumericComponents("v1.2-BETA1"),
+      ).toStrictEqual([1, 2, 0, 0, -2, 1]);
+    });
+  });
+
+  describe("compareVersions", () => {
+    it("should compare versions correctly", () => {
+      const expected = [
```

Comment:
If you use [`it.each`](https://jestjs.io/docs/api#1-testeachtablename-fn-timeout), it could be more transparent how many scenarios we're testing and it would display proper test name. This test however will be shown as a single test only. Not a huge deal though, but it would help troubleshooting a lot if you use `it.each` because it will treat each test as a separate test and you should know which one fails or passes rather than failing this test right away after the one of the case here failed. 

For example
https://github.com/metabase/metabase/blob/14e5d42c0cf2729be80b287051e056cfc8f86824/frontend/src/metabase/parameters/components/ValuesSourceSettings/ValuesSourceSettings.unit.spec.tsx#L27-L32